exercises/practice/say/say_test.cpp

```diff
@@ -5,93 +5,98 @@
+    REQUIRE("ninety-nine" == say::in_english(99));
 }
 
-TEST_CASE("one_hundred")
-{
-    REQUIRE("one hundred" == say::in_english(100ULL));
+TEST_CASE("one hundred", "[e417d452-129e-4056-bd5b-6eb1df334dce]") {
+    REQUIRE("one hundred" == say::in_english(100));
 }
 
-TEST_CASE("one_hundred_twenty_three")
-{
-    REQUIRE("one hundred twenty-three" == say::in_english(123ULL));
+TEST_CASE("one hundred twenty-three",
+          "[d6924f30-80ba-4597-acf6-ea3f16269da8]") {
+    REQUIRE("one hundred twenty-three" == say::in_english(123));
 }
 
-TEST_CASE("one_thousand")
-{
-    REQUIRE("one thousand" == say::in_english(1000ULL));
+TEST_CASE("two hundred", "[2f061132-54bc-4fd4-b5df-0a3b778959b9]") {
+    REQUIRE("two hundred" == say::in_english(200));
 }
 
-TEST_CASE("one_thousand_two_hundred_thirty_four")
-{
-    REQUIRE("one thousand two hundred thirty-four" == say::in_english(1234ULL));
+TEST_CASE("nine hundred ninety-nine",
+          "[feed6627-5387-4d38-9692-87c0dbc55c33]") {
+    REQUIRE("nine hundred ninety-nine" == say::in_english(999));
 }
 
-TEST_CASE("one_million")
-{
-    REQUIRE("one million" == say::in_english(1000ULL*1000ULL));
+TEST_CASE("one thousand", "[3d83da89-a372-46d3-b10d-de0c792432b3]") {
+    REQUIRE("one thousand" == say::in_english(1000));
 }
 
-TEST_CASE("one_million_two")
-{
-    REQUIRE("one million two" == say::in_english(1000ULL*1000ULL + 2ULL));
+TEST_CASE("one thousand two hundred thirty-four",
+          "[865af898-1d5b-495f-8ff0-2f06d3c73709]") {
+    REQUIRE("one thousand two hundred thirty-four" == say::in_english(1234));
 }
 
-TEST_CASE("one_million_two_thousand_three_hundred_forty_five")
-{
-    REQUIRE("one million two thousand three hundred forty-five" == say::in_english(1002345ULL));
+TEST_CASE("one million", "[b6a3f442-266e-47a3-835d-7f8a35f6cf7f]") {
+    REQUIRE("one million" == say::in_english(1000000));
 }
 
-TEST_CASE("one_billion")
-{
-    REQUIRE("one billion" == say::in_english(1000ULL*1000ULL*1000ULL));
+TEST_CASE("one million two thousand three hundred forty-five",
+          "[2cea9303-e77e-4212-b8ff-c39f1978fc70]") {
+    REQUIRE("one million two thousand three hundred forty-five" ==
+            say::in_english(1002345));
 }
 
-TEST_CASE("a_really_big_number")
-{
+TEST_CASE("one billion", "[3e240eeb-f564-4b80-9421-db123f66a38f]") {
+    REQUIRE("one billion" == say::in_english(1000000000));
+}
+
+TEST_CASE("a big number", "[9a43fed1-c875-4710-8286-5065d73b8a9e]") {
     REQUIRE(
         "nine hundred eighty-seven billion six hundred fifty-four million "
         "three hundred twenty-one thousand one hundred twenty-three" ==
-        say::in_english(987654321123ULL));
+        say::in_english(987654321123));
 }
 
-TEST_CASE("raises_an_error_below_zero")
-{
-    REQUIRE_THROWS_AS(say::in_english(-1), std::domain_error);
+TEST_CASE("numbers below zero are out of range",
+          "[49a6a17b-084e-423e-994d-a87c0ecc05ef]") {
+    REQUIRE("{'error': 'input out of range'}" == say::in_english(-1));
```

Comment:
It looks like the tool cannot handle cases with errors that well yet. For invalid input, we want the student to throw an exception.
```suggestion
    REQUIRE_THROWS_AS(say::in_english(-1), std::domain_error);
```